packages/api/cms-api/src/index.ts

```diff
@@ -70,10 +70,9 @@ export { defaultDamAcceptedMimetypes } from "./dam/common/mimeTypes/default-dam-
 export { DamConfig } from "./dam/dam.config";
 export { DAM_CONFIG, DAM_MODULE_OPTIONS, IMGPROXY_CONFIG } from "./dam/dam.constants";
 export { DamModule } from "./dam/dam.module";
-export { FileArgs } from "./dam/files/dto/file.args";
+export { DamScopeType } from "./dam/files/decorators/dam-scope-type.decorator";
```

Comment:
do you think that export name is understandable?

(should be done for Redirects/PageTree the same way, if we prefer this pattern)

Reply:
No, I don't like it very much. But I've decided to not add the same TODO once again 😁 

Problem is, it's both a input type as a object type as well. Maybe `DamScopeGraphQLType`?

Reply:
first, we have to think about weather this (new) pattern is useful and helpful.

if we decide no: just remove the comment
if we decide yes: find a good name

Reply:
What might be plausible alternatives?

1. Do nothing, and fail hard if the scope (or entity, in certain cases) has a different name
2. Validate the name in the library. Should be possible reading the metadata, but I've not tried it. However, may lead to breaking changes if Nest changes the decorators
3. Remove/ignore the decorators in the application code and wrap the entities in the library ourselves.

Alternative 3 would be the best (if it works), but alternative 2 might be okay as well.

Reply:
(3) won't work (as the class is app defined)

(2) sounds good!

Reply:
I've added the validation here: 2fc4e93c3658139bc07517f2b3a09f3fccb1ee55

> However, may lead to breaking changes if Nest changes the decorators

`TypeMetadataStorage` is part of Nest's public API, so this concern is no longer valid